### PR TITLE
New homepage featuring "The Feed"

### PR DESCRIPTION
**Pull request by zacjones93**
@joelhooks and I talked about simplifying the home page by removing curated sections and replace 

- [ ] TODO: update Sanity when this is approved

home page desktop
home mobile

### CODE REVIEW

File: src/components/app/header/index.tsx [hunk 1]
```diff
@@ -118,45 +118,6 @@ const navLinks = [
...
-    image: (props?: React.SVGProps<SVGSVGElement>) => (
-      <DocumentTextIconOutline
-        strokeWidth={1.5}
-        height={20}
-        width={20}
-        aria-hidden="true"
-        {...props}
-      />
-    ),
-  },
```
Comment (zacjones93) [inline src/components/app/header/index.tsx:159]: removed tips, articles, and guides from nav

File: src/components/pages/home/index.tsx [hunk 2]
```diff
@@ -38,20 +42,22 @@ const Home: React.FC<React.PropsWithChildren<any>> = ({
...
-            location={location}
-            completedCoursesIds={completedCourseIds}
-          />
-        </main>
-      </div>
+      <main className="pt-8 sm:pt-16">
+        <InstantSearchSSRProvider {...searchServerState}>
+          <TheFeed />
+        </InstantSearchSSRProvider>
+      </main>
```
Comment (zacjones93) [inline src/components/pages/home/index.tsx:60]: adds The Feed to the homepage and moves Jumbotron under main search

File: src/components/pages/home/the-feed.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                  )}
+                </main>
+              </div>
+            </div>
+          </div>
+        </InstantSearch>
+      </div>
+    </>
+  )
+}
```
Comment (zacjones93) [inline src/components/pages/home/the-feed.tsx:63]: The feed is just a simplified version of the instant search we have on `/q`

File: src/components/search/pagination.tsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import {Pagination as AlgoliaPagination} from 'react-instantsearch'
+import 'instantsearch.css/themes/algolia-min.css'
```
Comment (zacjones93) [inline src/components/search/pagination.tsx:3]: adding these styles makes pagination look decent

File: src/pages/index.tsx [hunk 5]
```diff
@@ -77,9 +87,14 @@ const homepageQuery = groq`*[_type == 'resource' && slug.current == "curated-hom
 export async function getStaticProps() {
   const data = await sanityClient.fetch(homepageQuery)
 
+  const searchServerState = await getServerState(<TheFeed />, {
+    renderToString,
+  })
+
```
Comment (zacjones93) [inline src/pages/index.tsx:93]: server render `TheFeed` component so there is no flash of no search results

**Review by Creeland — APPROVED** [review]

**Pull request merged**